portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +33,55 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
```

Comment:
You should update some of these comments to describe what the file is doing now that you have modified it.

Reply:
My bad, I didn't clean up comments. Same for the other four comments below.